example_configs/seafile.md

```diff
@@ -1,7 +1,9 @@
 # Configuration for Seafile
-Seafile's LDAP interface requires a unique, immutable user identifier in the format of `username@domain`. Since LLDAP does not provide an attribute like `userPrincipalName`, the only attribute that somewhat qualifies is therefore `mail`. However, using `mail` as the user identifier results in the issue that Seafile will treat you as an entirely new user if you change your email address through LLDAP. If this is not an issue for you, you can configure LLDAP as an authentication source in Seafile directly. A better but more elaborate way to use Seafile with LLDAP is by using Authelia as an intermediary. This document will guide you through both setups.
+Seafile can be bridged to LLDAP directly, or by using Authelia as an intermediary. This document will guide you through both setups.
+
+## Configuring Seafile (prior to v11.0) to use LLDAP directly
+Seafile's LDAP interface requires a unique, immutable user identifier in the format of `username@domain`. Since LLDAP does not provide an attribute like `userPrincipalName`, the only attribute that somewhat qualifies is therefore `mail`. However, using `mail` as the user identifier results in the issue that Seafile will treat you as an entirely new user if you change your email address through LLDAP. **Note that this isn't true starting Seafile v11.0 and ulterior versions.**
```

Comment:
Maybe move the note at the front:

Note for seafile before v11: ...

Reply:
Sorry for the delay, that's actually my first time playing with pull requests and change requests, so I waited for the right time to do it :).

I've reversed the two sections "priori to v11" and "v11+", and then put the note on the beginning on the section as you advised.